pkg/launcher/launcher.go

```diff
@@ -129,12 +130,24 @@ func Launch(params Parameters) {
 		os.Exit(1)
 	}
 
+	// Setup scheme for Grafana
+	if err := grafana.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "")
+		os.Exit(1)
+	}
+
 	// Setup all Controllers
 	if err := controller.AddToManager(mgr); err != nil {
 		log.Error(err, "")
 		os.Exit(1)
 	}
 
+	// TODO: should I worry about the context here?
+	if err := grafana.EnsureDashboardExported(context.Background(), namespace, mgr); err != nil {
```

Comment:
We didn't identify in what namespace we should create a Grafana CR during discussion at ZulipChat with @ryanemerson and @rigazilla . The `namespace` there is a watching namespace and can take following values
* single namespace - if operator installed and watching in one namespace
* multiple, comma separated values - if operator watching several namespaces
* empty - if operator watching entire cluster

In this case, we should install Grafana CR inside operatorNs or custom defined parameter namespace (via env var, for example). Also we should take care about potential Grafana CR orphan record, if operator will be uninstalled. The following scenario should be implemented: lookup Infinispan operator deployment itself the operator code and set it as controller with `controllerutil.SetControllerReference(...)`

Reply:
> We didn't identify in what namespace we should create a Grafana CR during discussion at ZulipChat with @ryanemerson and @rigazilla . The `namespace` there is a watching namespace and can take following values
> 
> * single namespace - if operator installed and watching in one namespace
> * multiple, comma separated values - if operator watching several namespaces
> * empty - if operator watching entire cluster
> 
> In this case, we should install Grafana CR inside operatorNs or custom defined parameter namespace (via env var, for example).

Yes, I thought it was the namespace the operator was installed in. I was thinking of just using the namespace the operator was installed in though, since we are expecting 1 operator eventually and would map well with having 1 dashboard. I will poke around unless you can point me to that :)

> Also we should take care about potential Grafana CR orphan record, if operator will be uninstalled. The following scenario should be implemented: lookup Infinispan operator deployment itself the operator code and set it as controller with `controllerutil.SetControllerReference(...)`

Yes, I thought about this before but then forgot about implementing it. I will have to find where this is.

Reply:
We have a function in the Infinispan controller code to lookup resources

Reply:
> Yes, I thought it was the namespace the operator was installed in. I was thinking of just using the namespace the operator was installed in though, since we are expecting 1 operator eventually and would map well with having 1 dashboard. I will poke around unless you can point me to that :)

I fear that install CR in the operator ns could be a problem for cluster scoped operator: this kind of operator runs in `openshift-operators` ns and I'm not sure we're allowed to install CR there